test/xapi/track_test.rb

```diff
@@ -68,32 +65,32 @@ def test_doc_format
 
   def test_track_with_gitter_room
     track = Xapi::Track.new('fruit', FIXTURE_PATH)
-
-    assert_equal "xfruit", track.gitter
+    assert_equal 'xfruit', track.gitter
   end
 
   def test_track_with_default_checklist_issue
     track = Xapi::Track.new('fruit', FIXTURE_PATH)
-
     assert_equal 1, track.checklist_issue
   end
 
   def test_custom_test_pattern
     track = Xapi::Track.new('fruit', FIXTURE_PATH)
-
-    files = [
-      "file.ext",
-      "test.ext",
-      "file.tst",
-      "sub/dir/file.ext",
-      "sub/dir/file.tst",
-    ]
-    assert_equal %w(file.tst sub/dir/file.tst), files.select {|f|
-      f =~ track.test_pattern
-    }
+    assert_equal(/\.tst$/, track.test_pattern)
```

Comment:
Ditto here. We should test something about the behavior of the class, not the literal test pattern.